src/components/UI/LoadingXl.tsx

```
@@ -0,0 +1,13 @@
+import React from "react";
+import { FiLoader } from "react-icons/fi";
+import UI from "../../styles/UI.module.scss";
+type LoadingXlProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)